tests/test_device.py

```diff
@@ -257,6 +257,8 @@ def parseBuildStamp(var, val):
 
         self.BuildStamp.addListener(parseBuildStamp)        
 
+        #self.add(pr.LocalVariable(name = 'TestArray[2]',value=0))
```

Comment:
Maybe delete the `#self.add(pr.LocalVariable(name = 'TestArray[2]',value=0))` and `#self.add(pr.LocalVariable(name = 'TestArray[2]',value=0))`?

Reply:
That is kind of my testing playground. I had those there to verify the overlap checking works write. One of these days I need to script a proper test for this feature.